firebase-installations-interop/src/main/java/com/google/firebase/installations/FirebaseInstallationsApi.java

```diff
@@ -51,4 +52,12 @@
    */
   @NonNull
   Task<Void> delete();
+
+  /**
+   * Register a listener to receive fid changes.
+   *
+   * @param listener implementation of the {@code FidListener} to handle fid changes.
+   * @hide
+   */
+  void registerFidListener(FidListener listener);
```

Comment:
Please add `@DeferredApi` annotation to this method. It's needed for proper dynamic module loading that is upcoming across Firebase SDKs.

Reply:
Also, it might make sense to add a `NonNull` annotation onto the listener argument.